index.html

```diff
@@ -5,27 +5,326 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Smashing Magazine</title>
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+HK&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
 </head>
-    <main>
-        <header>
-            <div class="header-container">
-                <figure>
-
-                </figure>
-                <div></div>
-                <div></div>
-                <div></div>
+    <div class="main-wrapper">
+        <div class="header">
+            <div class="nav-header">
+                <div class="figure"></div>
+                <div class="nav-item">
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
+                <div class="nav-item">
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
+                <div class="nav-item">
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
+                <div class="nav-item">
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
+                <div class="nav-item">
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
                 <div></div>
-                <div></div>
-                <a href=""></a>
             </div>
-        </header>
-        <section>
-            <section class="left-column"></section>
-            <section class="right-column"></section>
-        </section>
+        </div>
+        <div class="section">
```

Comment:
Each one of the 4 main articles should have an image in the top left corner with title of an article under it.